src/LiteCore/src/LiteCore.Shared/Interop/Fleece.cs

```diff
@@ -141,6 +141,55 @@ public static explicit operator FLSliceResult(FLSlice input)
         }
     }
 
+    internal unsafe partial struct FLSliceResult
+    {
+        public void* buf;
+        private UIntPtr _size;
+
+        public ulong size
+        {
+            get {
+                return _size.ToUInt64();
+            }
+            set {
+                _size = (UIntPtr)value;
+            }
+        }
+
+        public override int GetHashCode()
+        {
+            var hasher = Hasher.Start.Add(size);
```

Comment:
Is this whole thing basically just to get the toString method?

Reply:
.NET would like you to implement it if you implement Equals (you get a style warning otherwise).